mobile_verifier/src/heartbeats.rs

```diff
@@ -44,15 +44,15 @@ impl From<HeartbeatKey> for HeartbeatReward {
 
 pub struct HeartbeatDaemon {
     pool: sqlx::Pool<sqlx::Postgres>,
-    config_client: Client,
+    config_client: GatewayClient,
     heartbeats: Receiver<FileInfoStream<CellHeartbeatIngestReport>>,
     file_sink: FileSinkClient,
 }
 
 impl HeartbeatDaemon {
     pub fn new(
         pool: sqlx::Pool<sqlx::Postgres>,
-        config_client: Client,
+        config_client: GatewayClient,
```

Comment:
Similar to above suggestion be good to have all the internal instances of `config_client` renamed to `gateway_client`